### ISSUE TITLE
Spectating a player that ducks in 'practice mode' causes you to be ducked when you resume the game

### ISSUE THREAD

**Issue by checkcheckcheckcheck** [momentum-mod/game#1475]
**Repro:**

- Spectate another player in your lobby
- Other player activates 'practice mode'
- Other player activates 'duck'
- Resume your game by clicking X to quit spec
- Player is now 'duckbugged'

Concs will be thrown from lower in the player's model, jumps are muted etc.

**Fix:**

Pressing duck yourself corrects the behaviour.

**Video:**

[https://youtu.be/RnA29o-cO6k](url)

**Desktop:**

OS: Windows
Render: DX9 (all defaults, launched from system tray)
MM version: 0.9.0 (mapping mode OFF for both player and player being spectated)
UI: Default

To note, we tried this in RJ/SJ and it did not cause rockets or stickies to be fired from a ducking position.

**Comment by hexaflexahexagon** [comment]
i wonder if this is related to the savestate and duck interaction weirdness that we still see? i havent tried in conc but this sounds similar to me

**Comment by Kanisof** [comment]
In conc if you hold duck and then load a save state you are in a crouch state until you uncrouch, if that's what you're referring to.

**Comment by hexaflexahexagon** [comment]
> In conc if you hold duck and then load a save state you are in a crouch state until you uncrouch, if that's what you're referring to.

yeah that's what i was talking about, happens in every gamemode too

**Comment by Gocnak** [comment]
Does this still happen on latest 0.10?

**Comment by checkcheckcheckcheck** [comment]
just tested this in surf/conc with Natan and it would appear it's been fixed for free
